export const onError = (v1, v2, v3) => {
  throw new Error(v1)
}
